scripts/messages.js

```diff
@@ -58,12 +58,16 @@ let formatMessage = (message, callback) => {
     let action = formatting.getDataMask(message.text, /\+[^*\s]+/)
     let assignees = formatting.getDataMask(message.text, /@[^*\s]+/)
     let name = formatting.getName(message.user.name)
+    let poster = "@" + message.user.login + " ("+ name +")"
```

Comment:
We should switch using js string formatting instead of concatenating like this (probably not now, but after we lint the app)

Reply:
@zelima ok, good suggestion, I will do next `lint`